esm/interface/element.js

```diff
@@ -228,6 +229,9 @@ export class Element extends ParentNode {
     if (name === 'class')
       return this.className;
     const attribute = this.getAttributeNode(name);
+    if (!ignoreCase(this)) {
```

Comment:
this check is useless if `attribute` is `null` or `undefined` ... can we put it behind as suggested in my previous comment?

```js
return attribute && (ignoreCase(this) ? attribute.value : escape(attribute.value));
```

Thanks!